accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -568,14 +570,14 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                         </li>
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            Open and highlight all issues in the element, if any
+                           Any additional issues for the same element, if any, also open and get highlighted
```

Comment:
This sentence is the second of two things that happen then you select an element or issue, so it starts with the action word 'Open' .  The overall structure of this part of the guide is 
'Select ... to: 
- Highlight something 
- Open something'  

It doesn't work grammatically to start the second bullet with 'Any additional issues ...'
Also, the text has to apply both for the case where the user clicked on a single issue AND also when they selected an element that has multiple issues.  

So overall I feel the original text is better.